.github/ISSUE_TEMPLATE/decision.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Decision
+about: A binding record of a decision
+title: ''
+labels: kind/decision, lifecycle/needs-triage
+assignees: ''
+
+---
+<!--  Thanks for asking for a decision! Here are some tips for you:
+
+1. Decisions should be large ones, that cut across many parts of a repo
+   or the project as a whole. "" is an example of a decision that's relevant here
```

Comment:
Need to fill in the example decision here :)

Reply:
facepalm.jpg

Reply:
Fixed.